frontend/src/App.tsx

```diff
@@ -17,6 +18,7 @@ function App() {
         <Route path="/blog/:id" element={<Blog />} />
         <Route path="/publish" element={<Publish />} />
         <Route path="/edit/:id" element={<Edit />} />
+        <Route path="/userBookmarks" element={<UserBookmarks />} />
```

Comment:
Share the reasoning behind adding the prefix `User` to the Bookmarks component? 
Consider keeping the component as just <Bookmarks /> instead of <UserBookmarks />

Reply:
will rename to just Bookmarks!